result.go

```diff
@@ -0,0 +1,21 @@
+package sqrl
+
+import (
+	"database/sql"
+)
+
+func RowsAffected(res sql.Result, err error) (int64, error) {
```

Comment:
Please add a comment, especially that "`err` is an error from query execution". An example is also welcome :)